code/modules/cargo/goodies.dm

```diff
@@ -132,7 +132,11 @@
 	desc = "The most important thing you could possibly spend your hard-earned money on."
 	cost = PAYCHECK_EASY * 4
 	contains = list(/obj/item/toy/plush/beeplushie)
-
+/datum/supply_pack/goody/dyespray
```

Comment:
Please put an empty line before the type like the rest of the file, same for the one below this.